src/test/java/com/google/devtools/build/lib/view/java/JavaImportConfiguredTargetTest.java

```diff
@@ -190,18 +193,19 @@ public void testDeps() throws Exception {
         .containsExactly(
             "java/jarlib2/import.jar", "java/jarlib2/exportjar.jar", "java/jarlib2/depjar.jar")
         .inOrder();
-
-    // Recursive deps work the same as with java_library.
-    JavaCompilationArgsProvider compilationArgsProvider =
```

Comment:
This is identical to `recursiveCompilationArgs` declared earlier. Also remove a duplicate assertion.